### ISSUE TITLE
[bug] error discover Sentinel

### ISSUE THREAD

**Issue by namreg** [centrifugal/centrifugo#375]
**Describe the bug**
> A clear and concise description of what the bug is.

I have redis with sentinels deployed. Redis and each sentinel have a password.
When I try to deploy Centrifugo as described in docs, I see the errors:
```
{"level":"info","version":"2.5.1","runtime":"go1.14.3","pid":1,"engine":"Redis","gomaxprocs":12,"time":"2020-07-10T11:57:50Z","message":"starting Centrifugo"}
{"level":"info","path":"/centrifugo/centrifugo-config.json","time":"2020-07-10T11:57:50Z","message":"using config file"}
{"level":"warn","time":"2020-07-10T11:57:50Z","message":"secret is deprecated, use token_hmac_secret_key instead"}
{"level":"warn","time":"2020-07-10T11:57:50Z","message":"consider migrating to offset protocol field, details: https://github.com/centrifugal/centrifugo/releases/tag/v2.5.0"}
{"level":"info","time":"2020-07-10T11:57:50Z","message":"Redis: Sentinel for name: myapp, db: 1, using password: true"}
{"level":"debug","time":"2020-07-10T11:57:50Z","message":"running Redis PUB/SUB, num workers: 12"}
{"level":"error","error":"redigo: no sentinels available; last error: NOAUTH Authentication required.","time":"2020-07-10T11:57:50Z","message":"error loading Lua script"}
{"level":"debug","time":"2020-07-10T11:57:50Z","message":"stopping Redis PUB/SUB"}
{"level":"error","error":"redigo: no sentinels available; last error: NOAUTH Authentication required.","time":"2020-07-10T11:57:50Z","message":"error discover Sentinel"}
{"level":"error","error":"redigo: no sentinels available; last error: NOAUTH Authentication required.","time":"2020-07-10T11:57:51Z","message":"error flushing publish pipeline"}
{"level":"fatal","time":"2020-07-10T11:57:51Z","message":"error running node: publish control error, all shards failed: last error: redigo: no sentinels available; last error: NOAUTH Authentication required."}
```

Here are the options I use to start Centrifugo:
```
--config=/centrifugo/centrifugo-config.json
--health
--engine=redis
--redis_host=${REDIS_HOST}
--redis_password=${REDIS_PASSWORD}
--redis_master_name=${REDIS_MASTER_NAME}
--redis_sentinels=${REDIS_SENTINELS}
--redis_db=1
--port=8000
```

When I remove `redis_master_name` and `redis_sentinels ` than everything is working.

I convinced that specified password is correct.

**Versions**
> Centrifugo version is <???>
> Client library used is centrifuge-<???> of version <???>
> Operating system is <???>

...

**Steps to Reproduce**
> How can the bug be triggered?

…

**Expected behavior**
> What output or behaviour were you expecting instead?

…

**Code Snippets**
> A minimum viable code snippet can be useful! (use backticks to format it).

…

**Comment by FZambia** [comment]
Hello, Centrifugo only supports AUTH in Redis at moment, but not in Redis Sentinel. Will add a new option to set Sentinel password in next release.

**Comment by namreg** [comment]
Great, thanks!

**Comment by FZambia** [comment]
Released in v2.6.0
